  float64 leftMotorOutputRadPerSec
  float64 leftMotorFeedForwardVoltage
  float64 leftMotorAccelRadPerSec2
  float64 rightMotorOutputRadPerSec
  float64 rightMotorFeedForwardVoltage
  float64 rightMotorAccelRadPerSec2
  bool trajectoryActive
  bool trajectoryCompleted